BitwardenShared/UI/Vault/Vault/VaultList/VaultListState.swift

```diff
@@ -72,7 +72,7 @@ struct VaultListState: Equatable {
 
     /// Whether the import logins action card should be shown.
     var shouldShowImportLoginsActionCard: Bool {
-        importLoginsSetupProgress != nil && importLoginsSetupProgress != .complete
+        importLoginsSetupProgress != nil && importLoginsSetupProgress == .incomplete
```

Comment:
🎨 This is the same as comparing directly with `.incomplete`:
```suggestion
        importLoginsSetupProgress == .incomplete
```